### PR TITLE
SKALE-2810 compute public keys

### PR DESCRIPTION
**Pull request by olehnikolaiev**
add computing bls public key

### CODE REVIEW

File: .openzeppelin/project.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
     "SlashingTable": "SlashingTable",
     "Punisher": "Punisher",
     "TokenLaunchLocker": "TokenLaunchLocker",
-    "TimeHelpersWithDebug": "TimeHelpersWithDebug"
+    "TimeHelpersWithDebug": "TimeHelpersWithDebug",
```
Comment (DimaStebaev) [inline .openzeppelin/project.json]: Is is not needed in general case.

File: contracts/Schains.sol [hunk 2]
```diff
@@ -19,15 +19,20 @@
...
 pragma experimental ABIEncoderV2;
 
 import "./Permissions.sol";
 import "./SchainsInternal.sol";
 import "./ConstantsHolder.sol";
+import "./KeyStorage.sol";
 import "./SkaleVerifier.sol";
 import "./utils/FieldOperations.sol";
 
+interface ISkaleDKGCheck {
```
Comment (DimaStebaev) [inline contracts/Schains.sol]: Do we need this interface?

File: contracts/SkaleDKG.sol [hunk 3]
```diff
@@ -216,19 +203,17 @@ contract SkaleDKG is Permissions {
...
-                groupIndex,
-                channels[groupIndex].publicKey.x.a,
-                channels[groupIndex].publicKey.x.b,
-                channels[groupIndex].publicKey.y.a,
-                channels[groupIndex].publicKey.y.b
-            );
-            // delete channels[groupIndex];
-            channels[groupIndex].active = false;
-            emit SuccessfulDKG(groupIndex);
+            this.setSuccesfulDKG(groupIndex);
```
Comment (DimaStebaev) [inline contracts/SkaleDKG.sol]: Let's make `setSuccesfullDKG` public.

**Review by DimaStebaev — CHANGES_REQUESTED** [review]

**Review by payvint — APPROVED** [review]
OK!

**Review by DimaStebaev — APPROVED** [review]

**Pull request merged**